flow-typed/platform.js

```diff
@@ -1,7 +1,8 @@
 // @flow
-import type { Match } from 'react-router-dom';
```

Comment:
turns out you can't currently import a libdef into a libdef - this will just default to an `any` type